pages/trust-admin/wards/add.js

```diff
@@ -36,17 +35,24 @@ const AddAWard = ({ hospitals, error, hospitalId }) => {
 
 export const getServerSideProps = propsWithContainer(
   verifyTrustAdminToken(async ({ container, authenticationToken, query }) => {
-    const retrieveHospitalsByTrustId = container.getRetrieveHospitalsByTrustId();
-    const { hospitals, error } = await retrieveHospitalsByTrustId(
+    // const retrieveHospitalsByTrustId = container.getRetrieveHospitalsByTrustId();
+    // const { hospitals, error } = await retrieveHospitalsByTrustId(
+    //   authenticationToken.trustId
+    // );
```

Comment:
Is this code no longer needed?

Reply:
Yup. No longer need to retrieve hospitals because I have removed the hospitals dropdown box
Comments have been removed. 
All checks have passed. 🙂